doc/source/howtos/3nodesec2.rst

```diff
@@ -1,21 +1,38 @@
 .. _3nodec2:
 
-====================================================
-How to set up a three-node cluster in EC2 enviroment
-====================================================
+=====================================================
+How to set up a three-node cluster in EC2 environment
+=====================================================
 
-This manual assumes you are running *m1.xlarge* instances
-with Red Hat Enterprise Linux 6.1 64-bit.
+This manual assumes you are running three EC2 instances
+with Red Hat Enterprise Linux 7 64-bit.
 
 * ``node1``: ``10.93.46.58``
 * ``node2``: ``10.93.46.59``
 * ``node3``: ``10.93.46.60``
 
+.. rubric:: Recommendations on launching EC2 instances
+
+1. Select `instance types <https://aws.amazon.com/ec2/instance-types/>`_ that support Enhanced Networking functionality and offer instance storage via NVMe or SSD volumes.
+#. When adding instance storage volumes, choose the ones with good I/O performance:
+
+   * instances with NVMe are preferred
+   * GP2 SSD are preferred to GP3 SSD volume types due to I/O latency
+   * over sized GP2 SSD are preferred to IO1 volume types due to cost
+     
+#. Attach Elastic network interfaces with static IPs to your instances. 
+#. Assign Elastic IP addresses to your instances. This preserves your public IP addresses for other needs.
```

Comment:
```suggestion
#. Attach Elastic network interfaces with static IPs to your instances or Assign Elastic IP addresses to your instances. This preserves your IP addresses between instance restarts. It is needed because each PXC member should have `wsrep_cluster_address` option which points to IP addresses of both other members.
```